### ISSUE TITLE
Docs: Automatic deployment of website using Travis CI

### ISSUE THREAD

**Issue by Skaty** [PowerPointLabs/PowerPointLabs#1096]
Travis supports automatic deployment should the build passes on CI. I believe that we should automate the website updating process, so that the documentation can be up to date.

**Comment by damithc** [comment]
Note that docs need to match the features of the public release of the software.

**Comment by Skaty** [comment]
Agreed, which is why documentation and feature PRs should be merged together.

**Comment by damithc** [comment]
> Agreed, which is why documentation and feature PRs should be merged together

'merge' and 'release' are not the same. We 'merge' one PR at a time, and release in bulk at a later time.

**Comment by Skaty** [comment]
> 'merge' and 'release' are not the same. We 'merge' one PR at a time, and release in bulk at a later time.

Well, I meant merge to the branch that contains the release version. Alternatively, we could deploy based on tags, since that denotes particular releases of the software.

**Comment by damithc** [comment]
If it _can_ be automated, I'm all for it.

**Comment by Skaty** [comment]
Ok, I've been exploring possibilities to automate deployment of documentation using Travis CI.

It is possible and it can be done through Travis.

**Comment by damithc** [comment]
Good to hear that. At tomorrow's session you can explain exactly what you plan to automate and make a your case why it's worth the effort.

**Comment by YuPeiHenry** [comment]
@damithc What is the status of this issue?

**Comment by damithc** [comment]
> @damithc What is the status of this issue?

I don't remember what happened after that. @Skaty anything?

**Comment by Skaty** [comment]
Hmm, I explored but there needs to be a general cleanup of the Website repo.
